src/baldr/edgeinfo.cc

```diff
@@ -404,24 +404,24 @@ int8_t EdgeInfo::layer() const {
   return static_cast<int8_t>(value.front());
 }
 
-std::string EdgeInfo::level() const {
+std::vector<int16_t> EdgeInfo::level() const {
   const auto& tags = GetTags();
-  auto itr = tags.find(TaggedValue::kLevel);
-  if (itr == tags.end()) {
-    return "";
+  std::vector<int16_t> values;
+  for (auto [itr, range_end] = tags.equal_range(TaggedValue::kLevel); itr != range_end; ++itr) {
+    try {
+      values.emplace_back(static_cast<uint16_t>(std::stoi(itr->second)));
+    } catch (...) { LOG_ERROR("Unable to parse numerical level value " + itr->second); }
   }
-  const std::string& value = itr->second;
-  return value;
+  return values;
 }
 
-std::string EdgeInfo::level_ref() const {
```

Comment:
A comment about backwards compatibility:  this will lead to `level_ref()` returning single values (the first one it finds in the map) when there are multiple, but IMO the way this was previously handled also didn't make much sense. Maneuvers that end on a flight of stairs would have the instruction "Take the stairs to level 1;2".